testing/btest/scripts/base/files/x509/log-caching-cluster-zeromq.test

```diff
@@ -0,0 +1,76 @@
+# Test that certificate log deduplication works in clustered mode for ZeroMQ.
+#
+# @TEST-PORT: XPUB_PORT
+# @TEST-PORT: XSUB_PORT
+# @TEST-PORT: LOG_PULL_PORT
+#
+# @TEST-EXEC: cp $FILES/zeromq/cluster-layout-no-logger.zeek cluster-layout.zeek
+# @TEST-EXEC: cp $FILES/zeromq/test-bootstrap.zeek zeromq-test-bootstrap.zeek
+#
+# @TEST-EXEC: zeek --parse-only %INPUT
```

Comment:
Is this here for a reason, or is this a leftover from debugging? (Question repeated below, I noticed it first there)